dtfs-central-api/api-tests/v1/tfm/amendments/tfm-amendments-get.api-test.js

```diff
@@ -50,7 +46,7 @@ describe('GET TFM amendments', () => {
         .put({
           dealType: CONSTANTS.DEALS.DEAL_TYPE.BSS_EWCS,
           dealId,
-          auditDetails: generatePortalAuditDetails(MOCK_PORTAL_USER._id),
+          auditDetails: generateTfmAuditDetails(MOCK_TFM_USER._id),
```

Comment:
This is going to the submit to TFM endpoint so would always be a portal user. Not important since it's just test data though